spec/feature/user/edits_a_listing_spec.rb

```diff
@@ -14,7 +14,7 @@
     expect(page).to have_content("Edit listing")
   end
 
-  it "and is redirected to the edited listing and sees the edits" do
+  xit "and is redirected to the edited listing and sees the edits" do
     valid_user = create(:user)
```

Comment:
It was failing on friday we discussed. You and i discussed it and you said to set it as pending.